weblate/api/tests.py

```diff
@@ -2183,6 +2183,35 @@ def test_patch(self):
         self.assertEqual(Language.objects.get(code="cs").name, "New Language")
 
 
+class MemoryAPITest(APIBaseTest):
+    def test_get(self):
+        response = self.do_request(
+            "api:memory-list",
+            method="get",
+            superuser=True,
+            code=200,
+        )
+        self.assertEqual(response.status_code, 200)
+
+        response = self.do_request(
+            "api:memory-list",
+            method="get",
+            superuser=False,
+            code=403,
+        )
+        self.assertEqual(response.status_code, 403)
+
+    def test_delete(self):
+        response = self.do_request(
+            "api:memory-detail",
+            kwargs={"pk": 1},
+            method="delete",
+            superuser=True,
+            code=204,
+        )
+        self.assertEqual(response.status_code, 204)
```

Comment:
The do_request already checks the status code, so it is not needed again.
```suggestion
        self.do_request(
            "api:memory-list",
            method="get",
            superuser=True,
            code=200,
        )

        self.do_request(
            "api:memory-list",
            method="get",
            superuser=False,
            code=403,
        )

    def test_delete(self):
        self.do_request(
            "api:memory-detail",
            kwargs={"pk": 1},
            method="delete",
            superuser=True,
            code=204,
        )
```